user-service/controllers/user-controller.js

```diff
@@ -26,7 +26,7 @@ export async function loginUser(req, res) {
         // Generate access token
         const accessToken = generateAccessToken(user);
         console.log(accessToken)
-        res.cookie('accessToken', accessToken, { expires: new Date(Date.now() + (5 * 60 * 1000)), httpOnly: true }); // 5 minutes
+        res.cookie('accessToken', accessToken, { expires: new Date(Date.now() + (5 * 60 * 1000)), httpOnly: true, sameSite: 'none' }); // 5 minutes
```

Comment:
Can we put this variable in an env file somewhere?
so when we deploy to PROD it will take from the prod env file and set sameSite to Strict, but when we deploy to stag it will be set to none instead

Reply:
Ok sure makes sense